src/sage/topology/simplicial_complex.py

```diff
@@ -1248,13 +1351,23 @@ def _an_element_(self):
 
             sage: SimplicialComplex()._an_element_()
             ()
-            sage: simplicial_complexes.Sphere(3)._an_element_()
-            (0, 1, 2, 3)
+            sage: len(simplicial_complexes.Sphere(3)._an_element_())
+            4
```

Comment:
```suggestion
            sage: simplicial_complexes.Sphere(3)._an_element_()
            (0, 1, 2, 3)
```
The original test was better.